src/cpp/fastdds/domain/DomainParticipantImpl.cpp

```diff
@@ -830,9 +832,7 @@ PublisherImpl* DomainParticipantImpl::create_publisher_impl(
 bool DomainParticipantImpl::ignore_participant(
         const InstanceHandle_t& handle)
 {
-    static_cast<void>(handle);
-    EPROSIMA_LOG_ERROR(PARTICIPANT, "Not implemented.");
-    return false;
+    return rtps_participant_->ignore_participant(iHandle2GUID(handle).guidPrefix);
```

Comment:
This method should return a ReturnCode_t, and have the following implementation:

```suggestion
    return (nullptr == rtps_participant_) ? NOT_ENABLED :
        rtps_participant_->ignore_participant(iHandle2GUID(handle).guidPrefix) ? OK : ERROR;
```

Reply:
This has not been addressed